### ISSUE TITLE
Controller isn't creating ALB for EKS in existing VPC

### ISSUE THREAD

**Issue by woodmicha** [kubernetes-sigs/aws-load-balancer-controller#1771]
We're working on moving into EKS from a simple ALB->Target Group construct as we're about to scale up.

I was able to get everything working when I used the eksctl to create the cluster using defaults but that posed a problem in that our RDS is in an existing VPC.  I attempted doing it from the console and again doing with eksctl and specifying the existing vpc and subnets.  Everything applies w/o error but the net result is the controller never creates an ALB and the logging always times out when trying to see if there are any errors.

Any suggestions as to what I'm likely missing?

**Comment by kishorj** [comment]
@woodmicha, Did you add a new CIDR to your VPC after creating the k8s cluster? If you are using EKS, you can send us your cluster ARN and we can troubleshoot further.

**Comment by woodmicha** [comment]
Thanks for the reply Kishor.  I tried various things but don't think I
added new cidr's when I created clusters in our legacy VPC.  Maybe
eksctl did.  I ended up getting things working by peering the two VPCs (new
and legacy) and tweaking security groups.

I'll check to see if the cluster in question is still out there and if so
I'll FW you the ARN.  We were pressing hard to get done over the weekend so
we pivoted after the second try.

Appreciate the follow up.  Regards, Mike
